src/web/tsconfig.app.tsbuildinfo

```diff
@@ -0,0 +1 @@
+{"root":["./pages/App.tsx","./pages/AzureOptimizationEnginePage.tsx","./pages/BicepRegistryModulesPage.tsx","./pages/CostOptimizationWorkbookPage.tsx","./pages/FinOpsHubsPage.tsx","./pages/FinOpsWorkbooksPage.tsx","./pages/GovernanceWorkbookPage.tsx","./pages/HomePage.tsx","./pages/OpenDataPage.tsx","./pages/PowerBIReportsPage.tsx","./pages/PowerShellModulePage.tsx","./pages/ToolsLandingPage.tsx","./pages/main.tsx","./pages/__tests__/App.test.tsx","./pages/__tests__/HomePage.test.tsx","./components/__tests__/SideBar.test.tsx"],"version":"5.7.2"}
```

Comment:
Do we need to list out every page manually? This seems like it'll break at some point when we forget to add one 😕

Not sure if this is from this PR or another.

Reply:
Were doing some tests with build and vite preview, I think the TS compiler generated the file for incremental builds.

Reply:
Does the compiler always generate it? Should it be in .gitignore?